src/main/java/graphql/execution/AsyncExecutionStrategy.java

```diff
@@ -36,6 +36,7 @@ public AsyncExecutionStrategy(DataFetcherExceptionHandler exceptionHandler) {
     @Override
     @SuppressWarnings("FutureReturnValueIgnored")
     public CompletableFuture<ExecutionResult> execute(ExecutionContext executionContext, ExecutionStrategyParameters parameters) throws NonNullableFieldWasNullException {
+        executionContext.getDataLoaderDispatcherStrategy().executionStrategy(executionContext, parameters);
         Instrumentation instrumentation = executionContext.getInstrumentation();
```

Comment:
assign executionContext.getDataLoaderDispatcherStrategy() to a variable  and then use in all the places.

I feel its more readable